website/site/content/docs/add-to-your-site.md

```diff
@@ -220,7 +220,7 @@ collections:
 
 ## Authentication
 
-Now that you have your Netlify CMS files in place and configured, all that's left is to enable authentication. There are [many ways to do this](https://www.netlifycms.org/docs/authentication-backends) (with or without deploying to Netlify), but this example uses Netlify because it's one of the quickest ways to get started.
+Now that you have your Netlify CMS files in place and configured, all that's left is to enable authentication. Netlify is not required when setting up a backend, but this example uses Netlify because it's one of the quickest ways to get started. To use other authentication options or connect with other backends, see [authentication &amp; backends](https://www.netlifycms.org/docs/authentication-backends).
```

Comment:
@theetrain Thanks so much for this, it looks great! I would only suggest a change to this sentence:

> Netlify is not required when setting up a backend, but this example uses Netlify because it's one of the quickest ways to get started.

change to

> Netlify is not required to use the CMS, but this example uses Netlify for hosting and authentication because it's one of the quickest ways to get started.